src/components/OwnerPanel/OrderNow.tsx

```diff
@@ -0,0 +1,81 @@
+import React, { useState, useEffect, FC } from 'react';
+import classnames from 'classnames';
+import styles from './styles.module.scss';
+
+import DeliveryButton from './DeliveryButton';
+
+import { formatTime } from '../../utilities/general/textFormatter';
+
+interface Props {
+  showingAltLayout: boolean
+  hours: any[]
+  isMerchantOpen: boolean
+  deliveryService ?: any[]
+}
+
+const OrderNow: FC<Props> = ({ showingAltLayout, hours, isMerchantOpen, deliveryService }) => {
+  const [phone, setPhone] = useState<any | null>(null);
+  const [thirdPtyDelivery, setThirdPtyDelivery] = useState<any[]>([])
+
+  useEffect(() => {
+    if (deliveryService && deliveryService.length > 0) {
+      const copy = [...deliveryService]
+      const phoneIndex = deliveryService.findIndex((ele) => ele.phone_number)
+
+      copy.splice(phoneIndex, phoneIndex + 1)
+
+      setPhone(deliveryService[phoneIndex])
+      setThirdPtyDelivery(copy)
+    }
+  }, [deliveryService])
+
+  return (
+    <div className={classnames(styles.subsection, styles.orderNow)}>
+      {showingAltLayout ? <h2 className={classnames(styles.text__toggleMobileDropdown, styles.heading__mobileDropdown)}>View Hours and Order</h2> : null}
+      <div className={styles.orderNow__hoursSect}>
+        <p className={styles.boldText}>Hours: </p>
+        <div className={styles.orderNow__hours}>
+          {hours.length > 0
+            ? hours.map((ele) => {
+                if (ele.open_time === undefined) {
+                  return (
+                    <p className={styles.p__hours}>
+                      <span className={classnames(styles['textHours--bold'],styles.textHours)}>{ele.open_day}</span>
+                      <span className={classnames(styles.textHours, styles.textHour__hours)}>CLOSED</span>
+                    </p>
+                  );
+                }
+                const { open_day, open_time, close_time } = ele;
+                const openTime = formatTime(open_time);
+                const closeTime = formatTime(close_time);
+
+                return (
+                  <p className={styles.p__hours}>
+                    <span className={classnames(styles['textHours--bold'],styles.textHours)}>{open_day}</span>
+                    <span className={classnames(styles.textHours, styles.textHour__hours)}>
+                      {openTime + ' - ' + closeTime}
```

Comment:
nit: formatted strings might be easier / cleaner if you want them here, for this string, and the one a few lines below